apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,134 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      //this.sinon.useFakeTimers();
```

Comment:
I don't use `this.sinon.useFakeTimers()` in the setup because it breaks the throttle code in the tests that don't use `this.sinon.clock.tick()` by redefining Date and setTimeout. You are right I am going to remove the all setup function.